cranelift/filetests/filetests/isa/x86/bnot-b1.clif

```diff
@@ -0,0 +1,12 @@
+test compile
```

Comment:
Can you make this a binemit test instead? Also a run test may be appropriate.

Reply:
Done